src/Symfony/Component/Security/Core/Authentication/Token/TokenInterface.php

```diff
@@ -47,8 +48,8 @@ public function getCredentials();
     /**
      * Returns a user representation.
      *
-     * @return string|object Can be a UserInterface instance, an object implementing a __toString method,
-     *                       or the username as a regular string
+     * @return string|UserInterface Can be a UserInterface instance, an object implementing a __toString method,
```

Comment:
an object implementing `__toString()` is still valid as well. Or is this catched by "string"?

Reply:
fixed by listing `\Stringable`